src/services/car/leasing.ts

```diff
@@ -35,3 +36,35 @@ export const fetchLeasingFormUrl = async (args: {
       return null
   }
 }
+
+export const sendLeasingInterest = async (
+  listingId: number,
+  leasingInterest: LeasingInterest,
+  options = {}
+): Promise<WithValidationError<LeasingInterest>> => {
+  const { validateOnly, recaptchaToken } = {
+    validateOnly: false,
+    recaptchaToken: null,
+    ...options,
```

Comment:
What are you spreading the options for here if you only destructure the defined values anyways?

Reply:
Those are default values we want to use if nothing is provided.
If we have:
```javascript
const obj = { foo: "bar" }

const { foo } = { 
  foo: "default",
  ...obj,
}
```
`foo` would equal `"bar"` since the spread from `obj` overrides default.

Reply:
Mhh ofc misread that, thanks for the explanation